clang/lib/StaticAnalyzer/Checkers/MIGChecker.cpp

```diff
@@ -87,7 +90,7 @@ class MIGChecker : public Checker<check::PostCall, check::PreStmt<ReturnStmt>,
 #undef CALL
   };
 
-  CallDescription OsRefRetain{{"os_ref_retain"}, 1};
+  CallDescription OsRefRetain{CDM::SimpleFunc, {"os_ref_retain"}, 1};
```

Comment:
`CDM::SimpleFunc` is used instead of `CDM::CLibrary` because there might be some headers that declare `os_ref_retain` as a `static` and non-`inline` function -- e.g. random internet search provided [this source file](http://www.newosxbook.com/src.jl?tree=&file=/xnu-8792.81.2/libkern/os/refcnt.h).

Reply:
Yes, this is a plain C function. Definitely not builtin. May or may not come from system headers depending on what we're compiling.